docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -109,7 +109,7 @@ nef_cone(v::NormalToricVariety)
 dim(v::AbstractNormalToricVariety)
 dim_of_torusfactor(v::AbstractNormalToricVariety)
 euler_characteristic(v::AbstractNormalToricVariety)
-betti_number(v::AbstractNormalToricVariety, i::Int)
+betti_numbers(v::AbstractNormalToricVariety, i::Int)
```

Comment:
That renaming was *not* what I had in mind at all (though you are free to do it if you prefer). What I meant was that the vector with Betti numbers could be accessed via a helper with that name; since it is internal, perhaps prefix it by an underscore, so `_betti_numbers`

Reply:
Ok. Let me remove the last commit then.

I am uncertain if I understand your suggestion (both regarding the helper and `get_attribute!`, in the message below). Could you please elaborate more?